subsys/spm/secure_services.c

```diff
@@ -139,4 +143,17 @@ int spm_firmware_info(u32_t fw_address, struct fw_info *info)
 
 	return -EFAULT;
 }
-#endif
+#endif /* CONFIG_SPM_SERVICE_FIND_FIRMWARE_INFO */
+
+
+#ifdef CONFIG_SPM_SERVICE_PREVALIDATE
+__TZ_NONSECURE_ENTRY_FUNC
+int spm_prevalidate_b1_upgrade(u32_t dst_addr, u32_t src_addr)
+{
+	if (!bl_validate_firmware_available()) {
+		return -ENOENT;
```

Comment:
```suggestion
		return -ENOTSUP;
```
Just very minor suggestion, i believe it better reflect unsupported/disabled feature.